test.sh

```diff
@@ -40,15 +42,12 @@ do
 done
 echo "Main: ./test/Tests.ooc" >> "$TESTS_USE_FILE"
 rm -f .libs/tests-linux64.*
+
 rock -q +-Wall $ARGS $FLAGS $FEATURES $TESTS_USE_FILE
 exitcode=$?
-if [[ $exitcode -eq 0 && -f ./Tests ]]; then
+if [[ $exitcode -eq 0 && -f ./Tests ]]
+then
 	./Tests
 	exitcode=$?
 fi
-if [[ !( $exitcode == 0 ) ]]
-then
-	exit 1
-else
-	exit 0
-fi
+exit $exitcode
```

Comment:
How do you mean? `Tests.ooc` does not read output from `test.sh`, it's the other way around.